### PR TITLE
Make kvrocks2redis work

### PR DESCRIPTION
**Pull request by caipengbo**
I debugged kvRocks2Redis to make it work and added some content to make it easier to use.

### CODE REVIEW

File: tools/kvrocks2redis/config.h [hunk 1]
```diff
@@ -19,17 +19,19 @@ struct Config {
...
   std::string aof_file_name = "appendonly.aof";
   std::string next_offset_file_name = "last_next_offset.txt";
-  std::string next_seq_file_path = dir + "/last_next_seq.txt";
+  std::string next_seq_file_path = output_dir + "/last_next_seq.txt";
 
   std::string kvrocks_auth;
   std::string kvrocks_host;
   int kvrocks_port = 0;
   std::map<std::string, redis_server> tokens;
+  bool cluster_enable = true;
```
Comment (git-hulk) [inline tools/kvrocks2redis/config.h]: The default value should be false?

File: tools/kvrocks2redis/config.cc [hunk 2]
```diff
@@ -41,11 +41,25 @@ Status Config::parseConfigFromString(std::string input) {
       return Status(Status::NotOK, "argument must be 'yes' or 'no'");
     }
     daemonize = (i == 1);
-  } else if (size == 2 && args[0] == "dir") {
-    dir = args[1];
-    db_dir = dir + "/db";
-    pidfile = dir + "/kvrocks2redis.pid";
-    next_seq_file_path = dir + "/last_next_seq.txt";
+  } else if (size == 2 && args[0] == "data-dir") {
```
Comment (git-hulk) [inline tools/kvrocks2redis/config.cc:44]: this would have a breaking change for the old version?

File: tools/kvrocks2redis/config.h (hunk 1 above)
Comment (caipengbo) [reply tools/kvrocks2redis/config.h]: yeah

File: tools/kvrocks2redis/config.cc (hunk 2 above)
Comment (caipengbo) [reply tools/kvrocks2redis/config.cc:44]: I think it's a tool that won't be run by users for a long time, so making these changes doesn't affect users much, right?

Comment (git-hulk) [reply tools/kvrocks2redis/config.cc:44]: Okay, it's fine to me.

**Review by git-hulk — DISMISSED** [review]
LGTM, except the `cluster_enable` default value should be false.

File: tools/kvrocks2redis/config.h (hunk 1 above)
Comment (ShooterIT) [reply tools/kvrocks2redis/config.h]: `cluster_enable` should be false by default.

**Review by ShooterIT — CHANGES_REQUESTED** [review]

**Review by git-hulk — APPROVED** [review]

**Review by ShooterIT — APPROVED** [review]
LGTM, don't read

**Pull request merged**